samples/c/hello_nv12_input_classification_ov/main.c

```diff
@@ -104,10 +104,29 @@ void print_model_input_output_info(ov_model_t* model) {
  */
 
 bool is_supported_image_size(const char* size_str, size_t* width, size_t* height) {
-    char* p_end = NULL;
+    const char* _size = size_str;
     size_t _width = 0, _height = 0;
-    _width = strtoul(size_str, &p_end, 10);
-    _height = strtoul(p_end + 1, NULL, 10);
+    while (_size && *_size != 'x' && *_size != '\0') {
```

Comment:
Why replace strtoul with below code block?

Reply:
The problem comes that the returned width and height maybe be not assigned new value.